plugins/Live/Live.php

```diff
@@ -13,6 +13,7 @@
 use Piwik\API\Request;
 use Piwik\Common;
 use Piwik\Container\StaticContainer;
+use Piwik\Plugins\Live\Settings\VisitorLog;
```

Comment:
aliasing here might make it easier to read the code below as our namespaces are bad in conveying what the class actually does/represents. `VisitorLog::getInstance` below may confuse future us in terms of whether it is a setting or actual log feature that's getting instantiated.

```suggestion
use Piwik\Plugins\Live\Settings\VisitorLog as VisitorLogSettings;
```

Reply:
agree with the idea, but I've removed this line for this PR.